packages/frontend/src/components/Bid/BumpBid/BumpBidForm.tsx

```diff
@@ -32,8 +33,9 @@ export const BumpBidForm = ({
 }: BumpBidProps) => {
   const { account } = useEthers()
   const userBalance = useEtherBalance(account)
-  const notEnoughBalance = userBalance !== undefined && bumpAmount.gt(userBalance)
-  const bidTooLow = bumpAmount.lt(minimumIncrement)
+  const parsedBumpAmount = useMemo(() => parseEther(bumpAmount == '' ? '0' : bumpAmount), [bumpAmount])
```

Comment:
Does this need to be set in two places? It looks like it could be passed from `Flow` to `Form` through props.